applications/StructuralMechanicsApplication/custom_conditions/moving_load_condition.cpp

```diff
@@ -99,7 +99,10 @@ void MovingLoadCondition<TDim, TNumNodes>::InitializeSolutionStep(const ProcessI
     // check if cond should be calculated
     mIsMovingLoad = false;
     for (IndexType i = 0; i < TDim; ++i) {
-        if (std::abs(this->GetValue(POINT_LOAD)[i]) > std::numeric_limits<double>::epsilon() && local_x_coord <= this->GetGeometry().Length() && local_x_coord >= 0.0) {
+        if (std::abs(this->GetValue(POINT_LOAD)[i]) > std::numeric_limits<double>::epsilon() && 
+            local_x_coord <= this->GetGeometry().Length() + std::numeric_limits<double>::epsilon() && 
+            local_x_coord >= 0.0 - std::numeric_limits<double>::epsilon()) {
```

Comment:
These kinds of operations are so often used that it might make sense to pull them into a dedicated utility. I'll open a PR for one.

Reply:
in the mean time, could this fix be merged?

Reply:
Of course, sry. Got a bit disctracted and forgot about this one.

Reply:
thanks!